packages/rs-drive-abci/Cargo.toml

```diff
@@ -101,8 +101,8 @@ rocksdb = { version = "0.22.0" }
 integer-encoding = { version = "4.0.0" }
 
 [features]
-# TODO: Remove console for production
-default = ["mocks", "console"]
+
+default = ["mocks", "console", "grovedbg"]
```

Comment:
I think the idea to remove it from default

Reply:
ok